hooks/pre-commit

```diff
@@ -41,31 +41,13 @@ fi
 echo "${reset}"
 
 echo -ne "Checking for files that need boilerplate... "
-files_need_boilerplate=()
-boiler="${KUBE_HOOKS_DIR}/boilerplate.py"
-# Check for go files without the required boilerplate.
-if [[ ${#files[@]} -gt 0 ]]; then
-  files_need_boilerplate+=($("${boiler}" "go" "${files[@]}"))
-fi
-
-# Check for sh files without the required boilerplate.
-files=($(git diff --cached --name-only --diff-filter ACM | grep "\.sh" | grep -v -e "third_party" -e "Godeps"))
-if [[ ${#files[@]} -gt 0 ]]; then
-  files_need_boilerplate+=($("${boiler}" "sh" "${files[@]}"))
-fi
-
-# Check for py files without the required boilerplate.
-files=($(git diff --cached --name-only --diff-filter ACM | grep "\.py" | grep -v -e "third_party" -e "Godeps"))
-if [[ ${#files} -gt 0 ]]; then
-  files_need_boilerplate+=($("${boiler}" "py" "${files[@]}"))
-fi
-
-if [[ "${#files_need_boilerplate[@]}" -ne 0 ]]; then
+out=($(hack/verify-boilerplate.sh))
```

Comment:
Won't this catch files that may have snuck in but are not part of this commit?  Maybe not a huge deal, but we could pass "${files[@]}" to the script and have it use the commandline if present, else find files itself